src/batched/KokkosBatched_Util.hpp

```diff
@@ -43,6 +43,12 @@ namespace KokkosBatched {
 #pragma message "KOKKOS_BATCHED_DISABLE_AVX512"
 #endif
 
+#if defined(__FMA__)
+#pragma message "KOKKOS_BATCHED_ENABLE_FMA"
```

Comment:
@nmhamster @srajama1 @ambrad 

this is not about "not generating fma" but just "showing a message" that fma instruction is available. When benchmarking code, I would like if fma is available or not while compiling the code and this throw the message.